src/bot.ts

```diff
@@ -104,17 +130,93 @@ export default (app: Probot) => {
     context.log.info(`Posted comment about new claims: ${result.data.html_url}`);
   });
 
-  // Useful for testing purposes:
-  //
-  //  app.on('pull_request.closed', async (context) => {
-  //    const pullRequestNum = context.payload.pull_request.number;
-  //    const issueComment = context.issue({
-  //      body: `Thanks for CLOSING PR #${pullRequestNum}!`,
-  //    });
-  //
-  //    context.log.info(context.payload.repository.owner);
-  //
-  //    context.log.info('PR HAS BEEN CLOSED');
-  //    await context.octokit.issues.createComment(issueComment);
-  //  });
+  app.on("issue_comment.created", async (context: Context<'issue_comment.created'>) => {
+    const repo = context.payload.repository.name;
+    const owner = context.payload.repository.owner.login;
+    const organization = context.payload.organization?.login ?? "";
+    const sender = context.payload.sender.login;
+    const comment = context.payload.comment.body;
+    const issueNumber = context.payload.issue.number;
+    const issueCreatorId = context.payload.issue.user.id;
+    const htmlURL = context.payload.issue.html_url;
+
+    const isPR = htmlURL?.includes(`/pull/${issueNumber}`);
+
+    // check if comment sender is repo owner
+    if(owner !== sender) {
+      return;
+    }
+    // check if comment taggged gitpoap-bot
+    if(!comment.includes("@gitpoap-bot")) {
+      return;
+    }
+    // parse followed tagged contributors
+    const tagBody = comment.split("@gitpoap-bot ")[1];
+    const contributors = tagBody?.match(/@\w*/g)?.map(ele => ele.replace("@", "")).filter(ele => ele) ?? [];
+    const uniqueContributors =  Array.from(new Set(contributors));
+    // fetch github ids
+    const contributorGithubIds: number[] = [];
+    for(let contributor of uniqueContributors) {
+      const id: number = await fetch(`https://api.github.com/users/${contributor}`).then(res => res.json()).then(res => res.id);
```

Comment:
can you use octokit here instead pls. 

also in general, i'd rather we not use `.then` since we can use `async`/`await` in this repo